BootstrapKit/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,14 @@
+## Working Group Code of Conduct Template
+
+Every Working Group must have a Code of Conduct. By default,
+all repositories under the Node.js GitHub Organization fall under
+the [Node.js Code of Conduct][], but Working Groups can choose to
+define their own alternative policy. If the Working Group does choose
+it's own Code of Conduct, it must be documented in a `CODE_OF_CONDUCT.md`
+file located in the root of any repository under their stewardship.
```

Comment:
The CoC can't do less than the Node.js Code of Conduct. Or shouldn't be able to.

i.e This can't be used to bypass one, or if we overlooked that, we need to change things so it can't be.